panther_manager/include/panther_manager/plugins/action/signal_shutdown_node.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef PANTHER_MANAGER_SIGNAL_SHUTDOWN_NODE_HPP_
+#define PANTHER_MANAGER_SIGNAL_SHUTDOWN_NODE_HPP_
+
+#include <string>
+
+#include <behaviortree_cpp/action_node.h>
+#include <behaviortree_cpp/basic_types.h>
+#include <behaviortree_cpp/tree_node.h>
+
+namespace panther_manager
+{
+
+class SignalShutdown : public BT::SyncActionNode
+{
+public:
+  explicit SignalShutdown(const std::string & name, const BT::NodeConfig & conf)
+  : BT::SyncActionNode(name, conf)
+  {
+  }
+
+  static BT::PortsList providedPorts()
+  {
+    return {BT::InputPort<std::string>("message", "", "reason to shutdown robot")};
```

Comment:
Add `OutputPort<std::pair<bool, std::string>>("signal_shutdown")` to make to code more self exlpanatory